web/styles/portico/portico_signin.css

```diff
@@ -898,6 +898,8 @@ button#register_auth_button_gitlab {
 
     & fieldset {
         margin: 30px;
+        padding: 0;
+        border: 0;
```

Comment:
It looks like we also use this HTML element once on the `support_request` page; did you test that page?

```
 git grep fieldset
templates/corporate/support_request.html:            <fieldset class="support-request">
templates/corporate/support_request.html:            </fieldset>
templates/zerver/register.html:            <fieldset class="org-registration">
templates/zerver/register.html:            </fieldset>
templates/zerver/register.html:            <fieldset class="user-registration">
templates/zerver/register.html:            </fieldset>
web/styles/portico/portico_signin.css:    & fieldset {
```

Reply:
OK, I checked this and it turns out the `#registration` class is present on that page too; that's probably a copy/paste mistake but means this change is safe.

Reply:
I also checked it before opening the PR. Perhaps would have been better to mention it in the commit message.